recipes/newrelic/infrastructure/ohi/elasticsearch/windows.yml

```diff
@@ -0,0 +1,58 @@
+name: elasticsearch-open-source-integration
+displayName: Elasticsearch Open Source Integration
+description: New Relic install recipe for default Elasticsearch Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-elasticsearch
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: windows
+    kernelArch: x86_64
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - elastic
+  - elasticsearch
+
+# CLI runs process detection; this is used to filter recipes that are appropriate for matched processes
+processMatch:
+  - elasticsearch*
```

Comment:
@NRhzhao is there a way to tie up the regex so it matches more specifically the elasticsearch process? I assume it should with .exe at some point.

Reply:
@Julien4218  Updated process matching for specific executables, still works for both msi and choco install.  Also add a fallback to check elastic search port if can't find service running